go.mod

```diff
@@ -101,33 +101,33 @@ require (
 	github.com/ulikunitz/xz v0.5.12
 	github.com/vincent-petithory/dataurl v1.0.0
 	github.com/vmware/govmomi v0.37.2
-	golang.org/x/crypto v0.25.0
+	golang.org/x/crypto v0.27.0
 	golang.org/x/oauth2 v0.21.0
-	golang.org/x/sync v0.7.0
-	golang.org/x/sys v0.22.0
-	golang.org/x/term v0.22.0
+	golang.org/x/sync v0.8.0
+	golang.org/x/sys v0.25.0
+	golang.org/x/term v0.24.0
 	google.golang.org/api v0.189.0
 	google.golang.org/genproto/googleapis/api v0.0.0-20240711142825-46eb208f015d
 	google.golang.org/grpc v1.65.0
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.30.1
-	k8s.io/apiextensions-apiserver v0.30.1
-	k8s.io/apimachinery v0.30.1
-	k8s.io/client-go v0.30.1
+	k8s.io/api v0.30.3
+	k8s.io/apiextensions-apiserver v0.30.3
+	k8s.io/apimachinery v0.30.3
+	k8s.io/client-go v0.30.3
 	k8s.io/cloud-provider-vsphere v1.30.1
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.120.1
 	k8s.io/utils v0.0.0-20240310230437-4693a0247e57
 	libvirt.org/go/libvirtxml v1.10002.0
-	sigs.k8s.io/cluster-api v1.7.3
+	sigs.k8s.io/cluster-api v1.8.2
```

Comment:
We should also bump the capi binary to the same version in `cluster-api/cluster-api`. Building against one version and running another is asking for trouble.
When we bump, we need to make sure all the other providers still work, so this PR needs to be thoroughly tested.

Reply:
@r4f4 Updated...

Reply:
Weird it is here locally?
```
hamzy:hamzy-installer$ grep 'sigs.k8s.io/cluster-api v1.8.2' go.mod
	sigs.k8s.io/cluster-api v1.8.2
hamzy:hamzy-installer$ git status
On branch PowerVS-Update-417-CAPI-ibmcloud-c6bcd313
Your branch is up to date with 'origin/PowerVS-Update-417-CAPI-ibmcloud-c6bcd313'.
```

Reply:
Forgot to also modify `cluster-api/cluster-api/go.mod` :(
Thanks!